schemas/groups/electrical.json

```diff
@@ -184,7 +198,7 @@
       "properties": {
         "associatedBus": {
           "type": "string",
-          "description": "Name of BUS source is assocated with (if applicable, may = NULL)"
+          "description": "Name of BUS source is associated with (if applicable, may = NULL)"
```

Comment:
If a property is not applicable it can be omitted, as it is not required by the schema. I would remove the NULL note.